converters/toZigbee.js

```diff
@@ -26,8 +26,20 @@ const converters = {
     read: {
         key: ['read'],
         convertSet: async (entity, key, value, meta) => {
-            const result = await entity.read(value.cluster, value.attributes);
+            const result = await entity.read(value.cluster, value.attributes, (value.hasOwnProperty('options') ? value.options : {}));
             meta.logger.info(`Read result of '${value.cluster}': ${JSON.stringify(result)}`);
+            if (value.hasOwnProperty('store')) {
+                const state = {};
+                state[value.store] = result;
+                return {state: state};
```

Comment:
Can be simplified to

```js
return {state: {[value.store]: result}};
```

If I understand correctly, the goal is to expose the result on a specific property? Not sure if `store` is a good name.

Reply:
Yes, open to suggestions on the name. I’m not happy with store either but it was the best I could come up with to ‘store’ the result in the state.

Reply:
I called it 'state_property' instead, not sure it's a huge improvement. I did simplyfy the code.